src/services/plan-hierarchy.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  FreePath,
+  StandardPath,
+  PlusPath,
+  AgenciesPath,
+  PlanType,
+  Plan,
+  PathType,
+} from '../doppler-types';
+
+export interface PlanHierarchy {
```

Comment:
@andresmoschini This interface is almost empty, and implemented in `PlanService` file, shouldn´t it be better to have this all as part of `PlanService` file? https://github.com/FromDoppler/doppler-webapp/pull/1054/files#diff-b291d86559945d4b621d1612868d9d44R13

Reply:
Yes, I agree.